src/schemas/json/github-action.json

```diff
@@ -35,16 +35,17 @@
             "required": {
               "$comment": "https://help.github.com/en/github/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputsinput_idrequired",
               "description": "A boolean to indicate whether the action requires the input parameter. Set to true when the parameter is required.",
-              "type": "boolean",
-              "default": false
+              "type": "boolean"
             },
             "default": {
               "$comment": "https://help.github.com/en/github/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputsinput_iddefault",
-              "description": "A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file."
+              "description": "A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file.",
+              "type": "string"
             }
           },
           "required": [
-            "description"
+            "description",
+            "required"
```

Comment:
`required` is required, as documented on https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputsinput_idrequired

Reply:
@mathieudutour maybe it was in 2019 but right now, in 2020, that link specifically marks `reqiured` as an optional field.